app/controllers/items/new.js

```diff
@@ -114,6 +115,17 @@ export default GoodcityController.extend(
       }
     }),
 
+    canApplyDefaultValuation: Ember.computed("valueHkDollar", function() {
+      const defaultValue = this.get("defaultValueHkDollar");
+      const valueHkDollar = this.get("valueHkDollar");
+
+      if (!defaultValue) {
+        this.send("getItemValuation");
```

Comment:
Same comment as above, this modifies the state during a read.

Also note that `getXXX` is a standard naming format for getters, most people would assume it's a safe read.
Better to name it something like `updateValuation` which explicitly states that it will modify something

Reply:
thanks. its done